maven-model/src/main/mdo/maven.mdo

```diff
@@ -642,11 +642,11 @@
         </field>
       </fields>
     </class>
-    <class>
+    <class xdoc.anchorName="profile_build">
       <name>BuildBase</name>
       <version>3.0.0+</version>
       <superClass>PluginConfiguration</superClass>
-      <description>Generic informations for a build.</description>
+      <description>Build configuration in a profile.</description>
```

Comment:
Is this really only used in profiles?

Reply:
yes, this is what I discovered: the split between BuildBase and Build does not exist for some abstract reason to introduce class inheritance, but because BuildBase is used in profile, then define everything that can be overridden in profile and Build is the full main build element (with additional sourceDirectory, and so on)
I did not expect to discover that...